devel/qca/Portfile

```diff
@@ -1,223 +1,187 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
 
 PortSystem          1.0
-PortGroup           cmake 1.0
-PortGroup           active_variants 1.1
 
 name                qca
-version             2.1.0
-revision            3
```

Comment:
This is revision 3 and it should be kept further down for the old `qt4`-based port, no?!